dev/com.ibm.ws.security.jwtsso.token/resources/com/ibm/ws/security/jwtsso/token/resources/JWTSSOTOKENMessages.nlsprops

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.security/resources/com/ibm/ws/security/jwtsso/token/resources/JWTSSOTOKENMessages.nlsprops
+#COMPONENTPREFIX CWWKS
+#COMPONENTNAMEFOR WebSphere Application Server JWTSSOTOKEN
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+# Message prefix block: CWWKS6351 - CWWKS6399
+
+JWTSSO_CONFIG_INVALID=CWWKS6351E: The JWTSSO feature configuration is invalid. Check the JWTSSO feature configuration. Check for prior error messages in the log.
+JWTSSO_CONFIG_INVALID.explanation=A valid JWTSSO feature configuration could not be obtained.
+JWTSSO_CONFIG_INVALID.useraction=Check the JWTSSO feature configuration.  Check for prior error messages in the log.
+
+JWTSSO_CONFIG_INVALID_OR_TOKEN_INVALID=CWWKS6352E: JWTSSO token validation did not succeed. Check the JWTSSO feature configuration.  Check for prior error messages in the log.
+JWTSSO_CONFIG_INVALID_OR_TOKEN_INVALID.explanation=Validation of the supplied JWT token did not succeed.  The token or feature configuration might be invalid.
```

Comment:
Remove extra spaces between the two sentences on line 25.

Reply:
Will the user be familiar with the acronym "JWT"? (Usually we spell it out the first time it appears, but since I don't see messages in context, I don't know if it needs to be spelled out here  or not.)

Reply:
Would it be better to say, "...token failed."?